Source/JavaScriptCore/wasm/WasmB3IRGenerator.cpp

```diff
@@ -3148,9 +3149,10 @@ auto B3IRGenerator::addCallIndirect(unsigned tableIndex, const TypeDefinition& s
     return emitIndirectCall(calleeInstance, calleeCode, signature, args, results);
 }
 
-auto B3IRGenerator::addCallRef(const TypeDefinition& signature, Vector<ExpressionType>& args, ResultList& results) -> PartialResult
+auto B3IRGenerator::addCallRef(const TypeDefinition& originalSignature, Vector<ExpressionType>& args, ResultList& results) -> PartialResult
 {
     Value* callee = get(args.takeLast());
+    const TypeDefinition& signature = originalSignature.expand();
```

Comment:
`addCallRef` was adjusted to match `addCallIndirect` in doing the expansion here.